brilirs/Cargo.toml

```diff
@@ -31,7 +31,7 @@ itoa = "1.0"
 [dependencies.bril-rs]
 version = "0.1.0"
 path = "../bril-rs"
-features = ["ssa", "memory", "float", "speculate", "char"]
+features = ["memory", "float", "ssa", "speculate", "position", "import", "char"]
```

Comment:
I was surprised that these features weren't added since `brilirs` supports most of these. I realized that they probably get pulled in during feature resolution with `bril2json` which does list these features.

Reply:
That'd do it, I guess!